cfme/infrastructure/cluster.py

```diff
@@ -4,15 +4,15 @@
 :var page: A :py:class:`cfme.web_ui.Region` object describing common elements on the
            Cluster pages.
 """
+from functools import partial
```

Comment:
Should be a line break between the built-ins, thirdparty, and in-repo sets of imports